infra/base-images/base-runner/Dockerfile

```diff
@@ -14,41 +14,37 @@
 #
 ################################################################################
 
-# Build go stuff in its own image. We only need the resulting binaries.
-# Keeping the go toolchain in the image wastes 400 MB.
-FROM gcr.io/oss-fuzz-base/base-image as go-builder
+# Build rust stuff in its own image. We only need the resulting binaries.
+# Keeping the rust toolchain in the image wastes 1 GB.
+FROM gcr.io/oss-fuzz-base/base-image as temp-runner-binary-builder
 
-RUN apt-get update && apt-get install wget -y
-
-# Download and install the latest stable Go.
-ADD https://storage.googleapis.com/golang/getgo/installer_linux $SRC/installer_linux
-RUN chmod +x $SRC/installer_linux && \
-    SHELL="bash" $SRC/installer_linux && \
-    rm $SRC/installer_linux
-
-# Set up Golang environment variables (copied from /root/.bash_profile).
-ENV GOPATH /root/go
-
-# /root/.go/bin is for the standard Go binaries (i.e. go, gofmt, etc).
-# $GOPATH/bin is for the binaries from the dependencies installed via "go get".
-ENV PATH $PATH:/root/.go/bin:$GOPATH/bin
+RUN apt-get update && apt-get install -y \
```

Comment:
I'm sure some of this is unecesary, but it's not clear from @catenacyber what is needed.
I don't have time to figure this out right now and this extra stuff is discarded anyway, so the only effect is slowing down the build and maybe requiring more diskspace for the build while it's happening.

Reply:
I guess it is ok if you replaced curl with wget

Reply:
Sure. But I mean that we prob don't need to install python right?

Reply:
Not that I know, my patch for Rust coverage looked like
```
diff --git a/infra/base-images/base-runner/Dockerfile b/infra/base-images/base-runner/Dockerfile
index 3237916d..ee44768c 100755
--- a/infra/base-images/base-runner/Dockerfile
+++ b/infra/base-images/base-runner/Dockerfile
@@ -68,6 +68,7 @@ RUN apt-get update && apt-get install -y \
     python3 \
     python3-pip \
     wget \
+    curl \
     zip
```

Is this the reason for these changes ?

Reply:
I'm not sure you understand the purpose of the rust related parts of my patch.
The rust coverage patch installed the rust toolchain into base-runner. This adds 1 GB to the size of the image (more than doubling it).
I'm trying to fix this by installing the rust toolchain in a temporary image that does not become base-runner and gets deleted after building base-runner. The temporary image can build rustfilt with exploding the size of base-runner.
However, I don't know what dependencies are needed from apt to build rustfilt. For now I'm installing the same ones installed in base-runner, since I know these are sufficient. But there is extra stuff in here that we don't need.
I'm asking if you know which dependencies are actually needed to build rustfilt and which aren't.

Reply:
The curl removal was one thing we didn't need. But do we need python3 and python3-pip? What about zip? etc.

Reply:
Ok, I had not understood indeed.
I guess we just need `cargo` to install rustfilt...